package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azurefunctions",
     "displayName": "Azure Functions",
     "description": "%extension.description%",
-    "version": "0.6.0",
+    "version": "0.6.1",
```

Comment:
Do we need to update vscode engine to 1.20.0?

Reply:
Nope - this is still using the old versions of the ui/appservice packages